pkg/hub/submarineraddonagent/agent.go

```diff
@@ -59,30 +43,19 @@ const (
 	authenticatedGroup = "system:authenticated"
 )
 
-const agentInstallationNamespaceFile = "manifests/namespace.yaml"
-
-var agentDeploymentFiles = []string{
-	"manifests/serviceaccount.yaml",
-	"manifests/clusterrole.yaml",
-	"manifests/clusterrolebinding.yaml",
-	"manifests/role.yaml",
-	"manifests/rolebinding.yaml",
-	"manifests/deployment.yaml",
-}
-
 var agentHubPermissionFiles = []string{
-	"manifests/hub_role.yaml",
-	"manifests/hub_rolebinding.yaml",
+	"manifests/hub/hub_role.yaml",
+	"manifests/hub/hub_rolebinding.yaml",
```

Comment:
How about renaming these to avoid the `hub` stutter and match the `spoke` file names?